backend/src/schema/resolvers/posts.spec.js

```diff
@@ -1,458 +1,524 @@
-import { GraphQLClient } from 'graphql-request'
 import { createTestClient } from 'apollo-server-testing'
 import Factory from '../../seed/factories'
-import { host, login, gql } from '../../jest/helpers'
-import { neode, getDriver } from '../../bootstrap/neo4j'
+import { gql } from '../../jest/helpers'
+import { neode as getNeode, getDriver } from '../../bootstrap/neo4j'
 import createServer from '../../server'
 
 const driver = getDriver()
 const factory = Factory()
-const instance = neode()
-
-let client
-let userParams
-let authorParams
-
-const postId = 'p3589'
-const postTitle = 'I am a title'
-const postContent = 'Some content'
-const oldTitle = 'Old title'
-const oldContent = 'Old content'
-const newTitle = 'New title'
-const newContent = 'New content'
-const postSaveError = 'You cannot save a post without at least one category or more than three'
-const categoryIds = ['cat9', 'cat4', 'cat15']
-let createPostVariables
+const neode = getNeode()
 
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-03T10:40:54Z" title="Tuesday, September 3rd 2019, 12:40:54 pm +02:00">Sep 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,458 +1,524 @@
-import { GraphQLClient } from 'graphql-request'
 import { createTestClient } from 'apollo-server-testing'
 import Factory from '../../seed/factories'
-import { host, login, gql } from '../../jest/helpers'
-import { neode, getDriver } from '../../bootstrap/neo4j'
+import { gql } from '../../jest/helpers'
+import { neode as getNeode, getDriver } from '../../bootstrap/neo4j'
 import createServer from '../../server'
 
 const driver = getDriver()
 const factory = Factory()
-const instance = neode()
-
-let client
-let userParams
-let authorParams
-
-const postId = 'p3589'
-const postTitle = 'I am a title'
-const postContent = 'Some content'
-const oldTitle = 'Old title'
-const oldContent = 'Old content'
-const newTitle = 'New title'
-const newContent = 'New content'
-const postSaveError = 'You cannot save a post without at least one category or more than three'
-const categoryIds = ['cat9', 'cat4', 'cat15']
-let createPostVariables
+const neode = getNeode()
 
```


this is not an improvement, in my opinion...
I think the naming of these variables makes it clear what they are without need to read the actual string, and DRYs out the code by not have long strings repeated a hundred times.. there is a difference between repeating a relatively short variable name and a long string, but if it's an improvement for you, I'm willing to go with it.